.github/workflows/tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [ubuntu-latest, windows-latest, macos-latest]
-                node-version: [16.x, 18.x, 20.x]
+                node-version: [18.x, 18.x, 20.x]
```

Comment:
let's just delete this matrix altogether and replace line 21 with `node-version-file: '.nvmrc'`

Reply:
Actually this change is already on the main branch. You just need to rebase this one and discard this change

Reply:
I'm having trouble rebasing from main branch since the branch is protected and I can't push from that point

Reply:
I did manage to merge main branch to current branch not rebase though